src/Normalizer/JecouteSurveyNormalizer.php

```diff
@@ -41,6 +41,9 @@ public function normalize($object, $format = null, array $context = [])
 
     public function supportsNormalization($data, $format = null, array $context = [])
     {
-        return !isset($context[static::ALREADY_CALLED]) && $data instanceof Survey;
+        return !isset($context[static::ALREADY_CALLED])
+            && $data instanceof Survey
+            && 0 !== \count(array_intersect(['pap_campaign_read', 'survey_list'], $context['groups'] ?? []))
```

Comment:
```suggestion
            && 0 !== \count(array_intersect(['pap_campaign_read', 'phoning_campaign_read', 'survey_list'], $context['groups'] ?? []))
```

Reply:
we don't want the questions on phoning read. The reason why it appeared in the phoning test file is because we tested the sub-resource /phoning_campaign/{uuid}/survey